addons/web/static/src/views/debug_items.js

```diff
@@ -290,11 +291,12 @@ class SetDefaultDialog extends Component {
         }
         const fieldToSet = this.defaultFields.find((field) => {
             return field.name === this.state.fieldToSet;
-        }).value;
+        });
+        const valueToSet = mapWowlValueToLegacy(fieldToSet.value, fieldToSet.value.constructor.name.toLowerCase())
```

Comment:
```suggestion
      }).value;
      if(fieldToSet.constructor.name.toLowerCase() === "date"){
          fieldToSet = serializeDate(fieldToSet);
      } else if (fieldToSet.constructor.name.toLowerCase() === "datetime"){
          fieldToSet = serializeDateTime(fieldToSet);
      }
```
mapWowlValueToLegacy seems overkill for the situation and disappears in the later versions